digraph "unitGraph" {
    "r18 := @this: javaT.securityT.BasicPermission.NullOrEmptyName"
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(null)"
    "$r1 = new java.lang.Exception"
    "specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"
    "throw $r1"
    "$r15 := @caughtexception"
    "r19 = $r15"
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(null, \"value\")"
    "$r17 = new java.lang.Exception"
    "specialinvoke $r17.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"
    "throw $r17"
    "$r12 := @caughtexception"
    "r20 = $r12"
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(\"\")"
    "$r14 = new java.lang.Exception"
    "specialinvoke $r14.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"
    "throw $r14"
    "$r9 := @caughtexception"
    "r21 = $r9"
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"
    "$r11 = new java.lang.Exception"
    "specialinvoke $r11.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"
    "throw $r11"
    "$r6 := @caughtexception"
    "r22 = $r6"
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(null, \"value\")"
    "$r8 = new java.lang.Exception"
    "specialinvoke $r8.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"
    "throw $r8"
    "$r3 := @caughtexception"
    "r23 = $r3"
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"
    "$r5 = new java.lang.Exception"
    "specialinvoke $r5.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"
    "throw $r5"
    "$r2 := @caughtexception"
    "r24 = $r2"
    "return"
    "r18 := @this: javaT.securityT.BasicPermission.NullOrEmptyName"->"staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(null)";
    "r18 := @this: javaT.securityT.BasicPermission.NullOrEmptyName"->"$r15 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(null)"->"$r1 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(null)"->"$r15 := @caughtexception";
    "$r1 = new java.lang.Exception"->"specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")";
    "$r1 = new java.lang.Exception"->"$r15 := @caughtexception";
    "specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"throw $r1";
    "specialinvoke $r1.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"$r15 := @caughtexception";
    "throw $r1"->"$r15 := @caughtexception";
    "$r15 := @caughtexception"->"r19 = $r15";
    "r19 = $r15"->"staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(null, \"value\")";
    "r19 = $r15"->"$r12 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(null, \"value\")"->"$r17 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(null, \"value\")"->"$r12 := @caughtexception";
    "$r17 = new java.lang.Exception"->"specialinvoke $r17.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")";
    "$r17 = new java.lang.Exception"->"$r12 := @caughtexception";
    "specialinvoke $r17.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"throw $r17";
    "specialinvoke $r17.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"$r12 := @caughtexception";
    "throw $r17"->"$r12 := @caughtexception";
    "$r12 := @caughtexception"->"r20 = $r12";
    "r20 = $r12"->"staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(\"\")";
    "r20 = $r12"->"$r9 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(\"\")"->"$r14 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String)>(\"\")"->"$r9 := @caughtexception";
    "$r14 = new java.lang.Exception"->"specialinvoke $r14.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")";
    "$r14 = new java.lang.Exception"->"$r9 := @caughtexception";
    "specialinvoke $r14.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"throw $r14";
    "specialinvoke $r14.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"$r9 := @caughtexception";
    "throw $r14"->"$r9 := @caughtexception";
    "$r9 := @caughtexception"->"r21 = $r9";
    "r21 = $r9"->"staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(\"\", \"value\")";
    "r21 = $r9"->"$r6 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"->"$r11 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String getProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"->"$r6 := @caughtexception";
    "$r11 = new java.lang.Exception"->"specialinvoke $r11.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")";
    "$r11 = new java.lang.Exception"->"$r6 := @caughtexception";
    "specialinvoke $r11.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"throw $r11";
    "specialinvoke $r11.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"$r6 := @caughtexception";
    "throw $r11"->"$r6 := @caughtexception";
    "$r6 := @caughtexception"->"r22 = $r6";
    "r22 = $r6"->"staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(null, \"value\")";
    "r22 = $r6"->"$r3 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(null, \"value\")"->"$r8 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(null, \"value\")"->"$r3 := @caughtexception";
    "$r8 = new java.lang.Exception"->"specialinvoke $r8.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")";
    "$r8 = new java.lang.Exception"->"$r3 := @caughtexception";
    "specialinvoke $r8.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"throw $r8";
    "specialinvoke $r8.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected NullPointerException not thrown\")"->"$r3 := @caughtexception";
    "throw $r8"->"$r3 := @caughtexception";
    "$r3 := @caughtexception"->"r23 = $r3";
    "r23 = $r3"->"staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(\"\", \"value\")";
    "r23 = $r3"->"$r2 := @caughtexception";
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"->"$r5 = new java.lang.Exception";
    "staticinvoke <java.lang.System: java.lang.String setProperty(java.lang.String,java.lang.String)>(\"\", \"value\")"->"$r2 := @caughtexception";
    "$r5 = new java.lang.Exception"->"specialinvoke $r5.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")";
    "$r5 = new java.lang.Exception"->"$r2 := @caughtexception";
    "specialinvoke $r5.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"throw $r5";
    "specialinvoke $r5.<java.lang.Exception: void <init>(java.lang.String)>(\"Expected IllegalArgumentException not thrown\")"->"$r2 := @caughtexception";
    "throw $r5"->"$r2 := @caughtexception";
    "$r2 := @caughtexception"->"r24 = $r2";
    "r24 = $r2"->"return";
}
